### ISSUE TITLE
How can I use the model to recognize digit?

### ISSUE THREAD

**Issue by radeyang** [aymericdamien/TensorFlow-Examples#151]
I have downloaded your code ,and it works well.
Now I have a picture which was wrote a digit on ,I want to know 
how to use the model to recognize the digit?

**Comment by msampathkumar** [comment]
Check for TensorFlow-Examples/notebooks/3_NeuralNetworks/convolutional_network.ipynb

If you images are small enough like 28 by 28 and you think your system is good enough then you can try simple 3 layered Neural Network (linear) models.

Hope it helps.

**Comment by abdelouahabb** [comment]
@msampathkumar this might be a dumb question, but the result of the file is something to be plotted and visualized, not something a programmer (which doesent come to do stats) will use ? a programmer will use a result, so where are the values that the algorithm got after the iterations and optimized ? at the end, there is only `Optimization Finished! Accuracy: 0.9459` where are the values ? I mean, if someone wants to add the result of the library to OpenCV for example to make a complete image recognisation program, like Google Translator or a picture classifier, what he will use ?

Sorry for this question
